Why julia 2 (#12)
JuliaDataScience/JuliaDataScience: contents/1_preface.md

## Why Data Science? {#sec:why_data_science}

Data Science is not only machine learning and statistics, also it is not all about prediction, alas, it is not even a discipline fully contained within STEM (Science, Technology, Engineering, and Mathematics) fields [@Meng2019Data].
But one thing we can assure with highly confidence is that **data is always involved**.
Our aim in this book is two-fold:

* First, we want to focus on the backbone of data science: data.
* Second, we want you to use Julia.

We will cover why we think you should use Julia in Chapter [-@sec:why_julia].
For now, let's turn our attention towards data.

### Data is Everywhere {#sec:data_everywhere}

![Yes, data is the new black.](images/data-everywhere.png){#fig:data_everywhere}

**Data is abundant** and will be even more in the near future.
A report from the end of 2012 concluded that from 2005 to 2020, the digital universe will **grow by a factor of 300, from 130 exabyes^[1 exabyte (EB) = 1,000,000 terabyte (TB)] to a whooping 40,000 exabytes** [@gantz2012digital].
This is equal to 40 trillion gigabytes and, to put in perspective, more than **5,200 gigabytes for every living human currently in this planet!**

> In 2020, on average, every person created at least **1.7 MB of data per second** [@domo2018data]. A recent report predicted that almost **two thirds (65%) of national GDPs will will have undergone digitization by 2022** [@fitzgerald2020idc].

Reviewer: ```suggestion
In 2020, the average person created every person created **1.7 MB of data per second** [@domo2018data].
A recent report predicted that almost **two thirds (65%) of national GDPs will will have undergone digitization by 2022** [@fitzgerald2020idc].
```

> Every profession will be impacted not only by the increasing availability of data but also by data's increased importance [@chen2014big; @khan2014big].

Reviewer: Nice!

> This is why you should become competent in data skills.

Reviewer: Maybe, this could come across as a bit too harsh for some people. How about

```suggestion
This is why data skills are important.
```

Author: Yeah that was too harsh, but still doesn't total convey our message. We need a midpoint between what I've written and you propose. After all, I think a preface is sort of an "opinioned" text...
